index.html

```diff
@@ -64,5 +65,192 @@ <h2><a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connec
+            <li class="li-btn mobile-card">css</li>
+            <li class="li-btn mobile-card">javascript</li>
+          </ul>
+          <button id="pr-bt-3" class="project-btn">See Project</button>
+        </div>
+      </div>
+
+      <div class="project">
+        <img src="./images/work-sec-img/snapshot-4.svg" alt="Project image" class="project-image mobile-image" loading="lazy" width="641.2" height="478.183">
+        <div class="text-content">
+          <h3 class="title project-title mobile-card">Multi-Post Stories</h3>
+          <div class="project-data">
+            <!--mobile-->
+            <p class="bold mobile-card">CANOPY</p>
+            <p class="middot mobile-card">&bull;</p>
+            <p class="mobile-card">Back End Dev</p>
+            <p class="middot mobile-card">&bull;</p>
+            <p class="mobile-card">2015</p>
+          </div>
+          <p class="parrafo project-descr mobile-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="buttons-flex">  
+            <li class="li-btn mobile-card">html</li>
+            <li class="li-btn mobile-card">css</li>
+            <li class="li-btn mobile-card">javascript</li>
+          </ul>
+          <button id="pr-bt-4" class="project-btn">See Project</button>
+        </div>
+      </div>
+   </section>
+   <!-- works section (end) -->
+
+ <!-- about section (init)-->
+<section id="about-sec">
+  <div class="about-me">
+    <h2 id="tit-ab-me" class="title">About Myself</h2>   
+    <p id="intro-text" class="parrafo">
+      Hello, I'm a software developer! I can help you build a product,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>   
+    <h3><a href="https://linkedin.com/in/ivan-martinez-von-halle/" class="connect-text">LET'S CONNECT</a></h3>
+    <ul class="social-media-icons">
+      <li>
+        <a href="https://twitter.com/imprivado" aria-label="Twitter">
+          <img src="./images/first-page-img/twitter.svg" alt="twitter logo">
+        </a>
+      </li>
+      <li>
+        <a href="https://linkedin.com/in/ivan-martinez-von-halle/" aria-label="LinkedIn">
+          <img src="./images/first-page-img/linkedin.svg" alt="linkedin logo">
+        </a>
+      </li>
+      <li>
+        <a href="" aria-label="Medium">
+          <img src="./images/first-page-img/medium.svg" alt="medium logo">
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/ivanmvh" aria-label="GitHub">
+          <img src="./images/first-page-img/github.svg" alt="github logo">
+        </a>
+      </li>
+      <li>
+        <a href="" aria-label="Angel list">
+          <img src="./images/first-page-img/peace.svg" alt="Angel list logo">
+        </a>
+      </li>
+    </ul>
+    <button class="resume-btn">Get my resume</button>
+  </div>
+
+  <div class="about-languages">
```

Comment:
I suggest you match the template margins in this section also

- [ ] Your design

<img width="524" alt="Screenshot 2022-09-24 at 11 25 40" src="https://user-images.githubusercontent.com/48546992/192088562-69f3380b-acb0-4cb0-bea9-234957272411.png">

Figma Design
<img width="524" alt="Screenshot 2022-09-24 at 11 25 40" src="https://user-images.githubusercontent.com/48546992/192088598-c537c5a6-8099-42ce-a424-1ca2c908f331.png">